docker/install-yum.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -ex
+
+EPEL_RPM_HASH=0dcc89f9bf67a2a515bad64569b7a9615edc5e018f676a578d5fd0f17d3c81d4
+DEVTOOLS_HASH=a8ebeb4bed624700f727179e6ef771dafe47651131a00a78b342251415646acc
+
+# CentOS 5 is only available through vault.
+sed -i 's/enabled=1/enabled=0/' /etc/yum/pluginconf.d/fastestmirror.conf
+sed -i 's/mirrorlist/#mirrorlist/' /etc/yum.repos.d/*.repo
+sed -i 's|#\(baseurl.*\)mirror.centos.org/centos/$releasever|\1vault.centos.org/5.11|' /etc/yum.repos.d/*.repo
+sed -i 's|#\(baseurl.*\)download.fedoraproject.org/pub|\1archives.fedoraproject.org/pub/archive|' /etc/yum.repos.d/*.repo
+
+MY_DIR=$(dirname "${BASH_SOURCE[0]}")
+source "$MY_DIR/build-utils.sh"
+
+# EPEL support
+yum -y install wget curl
+# curl -sLO https://dl.fedoraproject.org/pub/epel/5/x86_64/epel-release-5-4.noarch.rpm
```

Comment:
What's the reason for bundling the binary rpm file in the repository?

Reply:
This is copied from the Python docker file, so I assume they had their own reasons for doing it (perhaps bandwidth related?). We can probably restore the download.